src/components/CertificateViewer.tsx

```diff
@@ -53,7 +53,8 @@ export const CertificateViewer = ({
       trace("reseting token information on unmount");
       resetTokenInformationState();
     },
-    [resetTokenInformationState, document]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It's usually a bad idea please explain

Reply:
For this useEffect, we need it to only run when the component unmounts with no deps. This is because the update/entry point of the document is using redux, whereas the token context is using react hooks, its not possible to use the reset method from context in the redux action, so i opted for a component complete dismount instead to reset the context state.

Reply:
running withe resetTokenInformationState as a dep would cause a loop where the reset would cause an update of callback and the useeffect would cause a reset

Reply:
this sounds fishy because the effect run only at unmount as thus changing `resetTokenInformationState` shouldn't trigger create a loop (nothing happens in the effect as is)
- are you sure about the observation ?
- ignoring the rule might create problem in the future (for instance if we add sth into the effect and forgot the dependency it wont be caught
- we should provide information directly in the code as comment to explain what this is doing)